.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
```

Comment:
This is where you should be adding content for jest and cypress as per the lessons